canonical-web-and-design/juju.is
# New homepage

## Done

Added new juju.is homepage, minus "what is juju" section (being worked on separately)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8041
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the page matches [the design](https://app.zeplin.io/project/606203889af57e12287b5308/screen/606edfa32c8c0b3ac8935e2c) and [the copy doc](https://docs.google.com/document/d/147spDkSVH0YaEGKZ4tDi7i28zzhIcacF1hBOMHvxaC0/edit#)


## Issue / Card

Fixes https://github.com/canonical-web-and-design/snap-squad/issues/2020
